packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -3471,12 +3471,49 @@ namespace Tpetra {
     TEUCHOS_TEST_FOR_EXCEPTION_CLASS_FUNC(
       ! hasColMap () || getColMap ().is_null (), std::runtime_error,
       "The graph must have a column Map before you may call this method.");
+    LocalOrdinal numLocalRows = this->getNodeNumRows ();
     TEUCHOS_TEST_FOR_EXCEPTION_CLASS_FUNC(
-      static_cast<size_t> (rowPointers.size ()) != this->getNodeNumRows () + 1,
+      static_cast<LocalOrdinal> (rowPointers.size ()) != numLocalRows + 1,
```

Comment:
If the number of local rows is zero, I think Tpetra permits `rowPointers` to have size zero.  This code doesn't change the current requirements, though.